spec/api/v1/external_clearstream_message_request_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require '/home/lex/Clients/Concur/Projects/stub_api/spec/helpers.rb'
```

Comment:
ditto

Reply:
Let's clean up the tests in the next feature branch, okay?